### DISCUSSION TITLE
API examples?

### DISCUSSION THREAD

**Discussion by brikim** [CyferShepard/Jellystat#275] [General]
Does anyone have any examples on how to use the api for Jellystat? I was wanting to add a widget possibly to homepage and can not find any examples. I see some of the api paths in the code but don't know how to set up the http request
http://jellystat.example.com/api/getHistory?key=???? . If anyone has any examples I would appreciate it.

Thanks!

**Comment by CyferShepard** [comment]
Hey @brikim , theres a swagger endpoint you can use to see the endpoints and examples, just go to `/swagger`

**Reply by brikim** [reply]
Doh didn't think of that! Thank you very much
